spec/integration/indirect_container_spec.rb

```diff
@@ -154,10 +154,9 @@ class FooHistory < ActiveFedora::Base
             it "has two related_objects" do
               expect(reloaded.related_objects).to eq [file, file2]
             end
-            it "has inbound triples" do
-              statement = file.reload.resource.query(predicate: ::RDF::URI.new('http://www.openarchives.org/ore/terms/proxyFor')).to_a.first
 
-              expect(statement.object).to eq file.resource.rdf_subject
+            it "has inbound triples" do
+              expect(Proxy.where(proxyFor_ssim: file.id).to_a).not_to be_empty
```

Comment:
Honestly, I would probably just remove this whole block, but I felt bad doing that. I'm not sure there really are inbound triples now. Any suggestions?